### PR TITLE
 ROI report has persistent inputs

### PR DESCRIPTION
**Pull request by ZitaNemeckova**
Fixes https://issues.redhat.com/browse/AA-850

- [x] show all/show button in table persists

- [x] manual cost of automation  persists

- [x] automated process cost persists

- [x] manual time input persists

### CODE REVIEW

File: src/QueryParams/useQueryParams.js [hunk 1]
```diff
@@ -193,6 +218,12 @@ const useQueryParams = (initial, namespace = DEFAULT_NAMESPACE) => {
         dispatch({ type: 'SET_LIMIT', value: limit });
       }
     },
+    setFromTable: (varName, value) => {
```
Comment (brumik) [inline src/QueryParams/useQueryParams.js:221]: I know this file is ripe for refactor, this is just my 2cents (take it or leave it):

These two functions are basically identical to using `dispatch({ type: 'ACTION', value: 'xxx' });` with an extra possibility of actionMapper not having a key. I think moving towards proper `redux styled actions` would be here more sensible:

```ts
// Extra typechecking and making sure the `type` always fits :)
const setManualCost = (value: number) => ({ type: 'SET_CALCULATOR_MANUAL', value });
...
// then in the calculator
dispatch(setManualCost(value));
```

Or just use dispatch with same complexity and more flexibility:
```ts
// calculator code:
dispatch({ type: 'SET_CALCULATOR_MANUAL' value });
```

**Comment by h-kataria** [comment]
@ZitaNemeckova can you please resolve conflicts on the PR

File: src/QueryParams/useQueryParams.js (hunk 1 above)
Comment (h-kataria) [reply src/QueryParams/useQueryParams.js:221]: @ZitaNemeckova I suggest going with the latter suggestion to be consistent. But we can  do that in a follow-up PR

**Comment by h-kataria** [comment]
@ZitaNemeckova I found an issue similar to https://issues.redhat.com/browse/AA-952 with Show/Hide toggle in this PR. You can fix that also in a follow-up PR or along with a fix for AA-952
Another issue that I found is that it keeps refreshing the screen as I am typing in input boxes to persist the changes, perhaps we can update the code to only re-render specific parts of the screen while the user is typing, like the chart/Total Savings component and Savings in the table. All this can be done in a separate PR, this PR is looking good as it is, so I am going go to go ahead and merge this one. Feel free to ping me if you need help or have questions.

**Review by h-kataria — APPROVED** [review]

**Pull request merged**